# 設問1の解答

## 環境

Python3.6

## 追加の仮定

故障状態のサーバは復活しないものとする．

## テストの内容

`surveilance_q1_01.csv`

故障しているサーバーが一つの場合のログ

`surveilance_q1_02.csv`

故障しているサーバが二つの場合のログ

`surveilance_q1_03.csv`

故障しているサーバが存在しない場合のログ

## テストの結果

3 passed
